sky/shell/switches.cc

```diff
@@ -20,6 +20,10 @@ const char kStartPaused[] = "start-paused";
 const char kTraceStartup[] = "trace-startup";
 const char kDeviceObservatoryPort[] = "observatory-port";
 const char kAotSnapshotPath[] = "aot-snapshot-path";
```

Comment:
Can we remove this one now that we have explicit switches for the individual files?